common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -149,6 +149,17 @@ class InheritanceMixin(XBlockMixin):
         default=True,
         scope=Scope.settings
     )
+    video_bumper = Dict(
+        display_name=_("Video Bumper Settings"),
+        help=_(
+            """Enter video bumper settings in the following format:"""
```

Comment:
I suggest, 
"Identify a video, 5-15 seconds in length, to play before course videos. Enter the video ID from the Video Uploads page and at least one transcript file in the following format: {"video_id": "ID", "transcripts": {"language": "filename.srt"}}. For example, an entry for a video with two transcripts looks like this:  {"video_id": "77cef264-d6f5-4cf2-ad9d-0178ab8c77be", "transcripts": {"en": "EDXDemoX-D01_1.srt", "ua": "EDXDemoX-D01_1_ua.srt"}}

Note that I can't quite tell from line 158 if you're looking for a language code (like en or ru) or a locale code (ua = Ukraine?). Let's adjust the text and get a good example based on which it is.